tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/JSDesugaring.scala

```diff
@@ -207,6 +293,7 @@ private[emitter] class JSDesugaring(internalOptions: InternalOptions) {
     // LHS'es for labeled expressions
 
     var labeledExprLHSes: Map[Ident, Tree] = Map.empty
+    val usedLabels = mutable.Set.empty[String]
```

Comment:
Forgot about this, sorry: Just to confirm, this means that label names need to be unique inside a method body. If so, we should probably update the IR checker to check this.